urbandict-search/flow-query.yml

```diff
@@ -3,6 +3,10 @@ with:
   read_only: true
   port_expose: 45678
 pods:
+  sentencizer:
```

Comment:
This is not needed, that is why we changed the `depth levels` at the rest of the Flow. The input of the search, will be a single sentence, so we do not want to `Sentencize`